JSyntaxPaneProxy is only needed for J5, we hardcode the path to the JSyntaxPane jar to /usr/share/java/jsyntaxpane.jar
--- a/freeplane_plugin_script/src/org/freeplane/plugin/script/JSyntaxPaneProxy.java
+++ b/freeplane_plugin_script/src/org/freeplane/plugin/script/JSyntaxPaneProxy.java
@@ -49,12 +49,15 @@
 		URL nodehighlighterJar;
 		try {
 			final URL pluginUrl = context.getBundle().getEntry("/");
+      /*
 			if (Compat.isLowerJdk(Compat.VERSION_1_6_0)) {
 				jsyntaxpaneJar = new URL(pluginUrl, "lib/jsyntaxpane/jsyntaxpane-jdk5.jar");
 			}
 			else {
 				jsyntaxpaneJar = new URL(pluginUrl, "lib/jsyntaxpane/jsyntaxpane.jar");
 			}
+      */
+      jsyntaxpaneJar = new URL("file:///usr/share/java/jsyntaxpane.jar");
 			nodehighlighterJar = new URL(pluginUrl, "lib/jsyntaxpane/nodehighlighter.jar");
 		}
 		catch (MalformedURLException e1) {
